fimsauthz-api/fims-authz.yaml

```diff
@@ -411,3 +423,12 @@ components:
             claims.  This is a string value consisting of the entire signed
             JWT.  A "signed_metadata" metadata value SHOULD NOT appear as a
```

Comment:
As per RFC 2119,  'SHOULD NOT' means not recommended.  Might we mean MUST NOT? http://microformats.org/wiki/rfc-2119

Reply:
This is verbatim from https://tools.ietf.org/html/rfc8414#section-2.1 .  Recommend leaving as-is.  Perhaps add that link here for clarity/reference to API implementers.